app/services/container_topology_service.rb

```diff
@@ -89,22 +89,26 @@ def entity_display_type(entity)
     end
   end
 
-  def build_entity_data(entity)
+  def entity_id(entity)
     type = entity_type(entity)
-    status = entity_status(entity, type)
-
     if entity.kind_of?(ManageIQ::Providers::ContainerManager)
       id = entity.id.to_s
```

Comment:
> I'm not sure how it's constructed but is the conversion to string required? Calling something an id that's a string is confusing in the rails world. I'm not opposed to it but it feels to me like the "port" as a String thing. In Rails, you'd expect ids can be sorted numerically among other things.

there's no sort needed here, and no new id string field is defined as part of this patch. there is a specific data structure built here for topology, and id is just used as part of it.
also, the id.to_s was not added as part of this patch, so I'm unclear why is this part of this particular change review.